charts/events/templates/event-sources.yaml

```
@@ -0,0 +1,25 @@
+{{- if index .Values "argo-events" "enabled" }}
+  {{- range $eventSource := $.Values.eventSources }}
+    {{- if eq $eventSource.name "webhook" }} 
+---
+apiVersion: argoproj.io/v1alpha1
+kind: EventSource
+metadata:
+  name: webhook
+spec:
+  service:
+    ports:
+      - port: {{$eventSource.port }}
+        targetPort: {{$eventSource.port }}
+  webhook:
+    # event-source can run multiple HTTP servers. Simply define a unique port to start a new HTTP server
+    example:
+      # port to run HTTP server on
+      port: "{{$eventSource.port }}"
+      # endpoint to listen to
+      endpoint: /{{ $eventSource.endpoint }}
+      # HTTP request method to allow. In this case, only POST requests are accepted
+      method: POST
```

Perhaps something like this?
```suggestion
  {{- if $eventSource.webhook }}
  webhook:
    {{- $eventSource.webhook | toYaml | nindent 6 }}
  {{- end }}
```